src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java

```diff
@@ -1044,7 +1044,8 @@ private void findBombardOptions(final PlayerID player, final List<Territory> myU
       // Check strafing and using allied attack if enemy capital/factory
       boolean isEnemyCapitalOrFactory = false;
       final TerritoryAttachment ta = TerritoryAttachment.get(t);
-      if ((ta != null && ta.isCapital()) || ProMatches.territoryHasInfraFactoryAndIsLand(player).match(t)) {
+      if (!t.getOwner().isNull()
```

Comment:
Sorry this is technically not a null check. This is to ensure the factory has a non-neutral owner (!t.getOwner.isNull()) as having the AI strafe neutral factories was wasting a large deal of units. This should probably have been in a separate commit but was quickly making these while I was testing the AI on a few maps.